Issue on Zeit now.sh

Call stack:
```
FATAL  Cannot find module 'chokidar'
Require stack:
- /var/task/node_modules/@nuxtjs/svg-sprite/lib/module.js
- /var/task/node_modules/@nuxt/core/dist/core.js
- /var/task/now__launcher.js
- /var/runtime/UserFunction.js
- /var/runtime/index.js
  Error: Cannot find module 'chokidar'
  Require stack:
  - node_modules/@nuxtjs/svg-sprite/lib/module.js
  - node_modules/@nuxt/core/dist/core.js
  - now__launcher.js
  - /var/runtime/UserFunction.js
  - /var/runtime/index.js
  at Object.<anonymous> (node_modules/@nuxtjs/svg-sprite/lib/module.js:1)
2020-03-08T16:41:14.031Z	undefined	ERROR	λ Error while initializing nuxt: /var/task/node_modules/@nuxtjs/svg-sprite/lib/module.js:1
```

--- a/package.json
+++ b/package.json
@@ -42,6 +42,7 @@
   },
   "dependencies": {
     "chalk": "^3.0.0",
+    "chokidar": "^3.3.1",
     "consola": "^2.11.3",
     "fs-extra": "^8.1.0",
     "svgo": "^1.3.2",

